next.config.js

```diff
@@ -200,17 +198,17 @@ const sentryWebpackPluginOptions = {
 
   // For all available options, see:
   // https://github.com/getsentry/sentry-webpack-plugin#options.
-};
 
-const sentryOptions = {
   // Upload additional client files (increases upload size)
   // https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/#widen-the-upload-scope
   widenClientFileUpload: true,
   hideSourceMaps: false,
+
+  sourcemaps: {
+    disable: process.env.UPLOAD_SENTRY_SOURCEMAPS !== "true",
+  },
 };
 
-export default withSentryConfig(
-  nextConfig,
-  sentryWebpackPluginOptions,
-  sentryOptions,
-);
+export default withSentryConfig(nextConfig, {
+  ...sentryOptions,
+});
```

Comment:
I don't think this is needed, or is it?

```suggestion
export default withSentryConfig(nextConfig, sentryOptions);
```